### PR TITLE
7 new dlls, x64 tested only

### PR DESCRIPTION
**Pull request by galoryber**
Only adding a handful of DLLs that weren't in the project at all, only tested x64.

### CODE REVIEW

File: yml/microsoft/built-in/textshaping.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+Name: textshaping.dll
+Author: Gary Lobermier
+Created: 2023-05-22
+Vendor: Microsoft
+ExpectedLocations:
+  - '%SYSTEM32%'
+ExpectedSignatureInformation:
```
Comment (wietze) [inline yml/microsoft/built-in/textshaping.yml:8]: Just double checking, does this relate to the DLL?

File: yml/microsoft/external/symsrv.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+Name: symsrv.dll
+Author: Gary Lobermier
+Created: 2023-05-22
+Vendor: Microsoft
+ExpectedLocations:
+  - '%PROGRAMFILES%\Windows Kits\10\Debuggers\x64'
```
Comment (wietze) [inline yml/microsoft/external/symsrv.yml:7]: See above

**Review by galoryber — COMMENTED** [review]
Agreed with all changes, updated locations, changed to variable locations, and added signature information as it applies to DLLs. I didn't make mention of signature information on the vulnerable EXEs, since those are unchanged in the attack, but the DLL should usually have the signature.

**Review by wietze — APPROVED** [review]
Superb, thanks for your contribution!

**Pull request merged**